tests/taskcluster/bin/raptor_graph

```diff
@@ -23,6 +23,9 @@ var originalHeadRev = process.env.GITHUB_HEAD_REV;
 
 const TC_CLIENT_OPTS = { timeout: 30 * 1000 };
 
+// taskcluster worker type for raptor tasks
+RAPTOR_WORKER_TYPE = 'raptor-single-capacity';
```

Comment:
Hrm, I'm not sure what you mean by a differnet line on TH.  the worker type doesn't dictate where it shows up on TH, that's based on platform I think.  *shrug* whatever works for me :)